src/Symfony/Component/Lock/Tests/Store/StoreFactoryTest.php

```diff
@@ -58,12 +58,14 @@ public function validConnections()
         }
         if (class_exists(\Memcached::class) && class_exists(AbstractAdapter::class)) {
             yield ['memcached://server.com', MemcachedStore::class];
+            yield ['memcached:?host[localhost]&host[localhost:12345]', MemcachedStore::class];
         }
-        if (class_exists(\Redis::class) && class_exists(AbstractAdapter::class)) {
+        if ((class_exists(\Redis::class) || class_exists(\Predis\Client::class)) && class_exists(AbstractAdapter::class)) {
```

Comment:
Revert this.

The purpose of this check is to avoid failing tests when the extension is not installed.

By checking if Predis is installed (which is always true because in require-dev) this breaks the initial check.

Reply:
fixed in 6a79f3e06fb9560ed3ec9f57983415a50e883887